### PR TITLE
feature crear vista perfil & iconos perfil, productos

### LINKED ISSUE #138: CREAR VISTA PARA LOS DATOS DEL PERFIL DE PROVEEDOR
TASK CREAR VISTA PARA LLENAR LOS DATOS DEL PERFIL DE PROVEEDOR

### EXPLANATION

Al entrar con usuario y contraseña de proveedor
se abre una vista de pantalla con los siguientes datos a pedir 
primero arriba el botón : regresar  que regresa a la vista anterior donde  pide usuario y contraseña
segundo  logotipo de la empresa: tendrá un espacio para colocar el logo de la empresa
tercero nombre de la marca: tendrá un espacio para incluir el nombre de la marca de la empresa
cuarto link del catalogo : tendrá espacio para incluir el link del catalogo de la empresa
quinto las opciones  cancelar y guardar : función guardar  guarda la información y funcion eliminar elimina la información

 ### SCREENSHOTS 
1 Entro con usuario y contraseña de proveedor

Image

2 Crear esta vista

Image

### USER STORY

Yo como usuario proveedor quiero que al entrar a crear mis datos de perfil en la parte superior quiero que este la opción regresar

Yo como usuario proveedor quiero que  al entra a ingresar mis datos  este un espacio para el logo de mi empresa

Yo como usuario proveedor quiero que al ingresar mis datos de perfil este un espacio  para el nombre de la marca

Yo como usuario proveedor quiero que aya un espacio para incluir el numero de whatsapp

Yo como usuario proveedor quiero que al ingresar mis datos aya un espacio para el link del catalogo 

 Yo como usuario proveedor quiero que al ingresar mis datos están  las opciones guardar y cancelar

### ACEPTANCE CRITERIAN
En esta vista se pedirán los datos  para el perfil de la empresa

# Referencias

- Por favor terminar en 3 días. Mas información: https://github.com/Azordev/did-admin-panel/issues/192

### LINKED ISSUE #140: (TASK)CREAR PERFIL LOGO DEL PERFIL Y LOGO DE PRODUCTO
TASK 
### EXPLANATION 
 Después de haber usado usuario y contraseña en el panel de DID se abre la siguiente vista donde se muestra una barra lateral con los logo perfil y  productos ,esta vista es exclusiva solo para proveedores ya ingresados y se pedirá  al lado una serie de datos para ser incluido con un perfil de empresa para incluir 

### SCREENSHOTS

1 Incluir nombre de usuario proveedor y contraseña

Image

2 Se desplegara la siguiente vista de pantalla

Image

3 hacer logo de  perfil  y logo de productos de la barra lado izquierdo

Image

### CHECKLIST

El usuario proveedor  tendrá acceso a este perfil para  ingresar su empresa a la plataforma

### REFERENCE
 En este link puedes conseguir logos
https://es.dreamstime.com/illustration/perfil.html

# Referencias

- Por favor terminar en 3 días. Mas información: https://github.com/Azordev/did-admin-panel/issues/192

### PR DESCRIPTION
**Pull request by JuanParraIV**
### Description

Please include a summary of the change.

fixes # (#140 #138)

#### Screenshots
#140 
image

#138 
image

If applicable, add screenshots about your changes

#### Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How to test it

If applicable, describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

### Checklist:

- [ ] Doesn't break the current code.
- [ ] Passes linters and test, also is building.
- [ ] Doesn't have spelling or grammatical problems.
- [ ] Doesn't have unnecessary comments or debugging code.
- [ ] The branch is updated with the last dev branch changes.

### CODE REVIEW

File: components/ProfileForm/ProfileForm.module.scss [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  overflow: hidden;
+  position: absolute;
+  width: 0.1px;
+  z-index: -1;
+}
+
+.label,
+.label-title {
+  color: $gray-300;
+  font-family: 'Visby', sans-serif;
```
Comment (Israel-Laguan) [inline components/ProfileForm/ProfileForm.module.scss]: Please check if this is a better option: `$font-visby, $font-base-sans-serif-family`

File: containers/Perfil/ProfileContainers.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+const ProfileContainers = () => {
+  return (
+    <div>
+      <BackHeader />
+      <ProfileForm />
+    </div>
+  )
+}
+
```
Comment (Israel-Laguan) [inline containers/Perfil/ProfileContainers.tsx]:
```suggestion
const ProfileContainers = () => (
    <div>
      <BackHeader />
      <ProfileForm />
    </div>
  )

```

Note it still needs indentation fix

File: pages/perfil/index.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+import ProfileContainers from 'containers/Perfil/ProfileContainers'
+
+const Profile: NextPage = () => {
+  return (
+    <ClientOnly>
+      <ProfileContainers />
+    </ClientOnly>
+  )
+}
```
Comment (Israel-Laguan) [inline pages/perfil/index.tsx]:
```suggestion
const Profile: NextPage = () => (
    <ClientOnly>
      <ProfileContainers />
    </ClientOnly>
  )
```

Note it still needs indentation fix

**Review by Israel-Laguan — APPROVED** [review]

File: containers/Perfil/ProfileContainers.tsx [hunk 4]
```diff
@@ -1,13 +1,10 @@
...
 import ProfileForm from '@/components/ProfileForm'
 
-const ProfileContainers = () => {
-  return (
+const ProfileContainers = () => (
     <div>
       <BackHeader />
       <ProfileForm />
     </div>
   )
```
Comment (Israel-Laguan) [inline containers/Perfil/ProfileContainers.tsx]: Check indentation

**Review by Israel-Laguan — APPROVED** [review]

**Comment by Eghinner** [comment]
@angelik0828 what do you think of the change in the back button?
chrome_YjyZd0NM4j

File: components/CustomInput/CustomInput.module.scss [hunk 5]
```diff
@@ -1,11 +1,14 @@
...
+.custom-input {
   margin-bottom: 23px;
 }
-.labelInput {
-  font-size: 30px;
+
+.label-input {
   color: $gray-850;
+  font-size: 30px;
 }
```
Comment (Eghinner) [inline components/CustomInput/CustomInput.module.scss]: restore to camelCase and implement `.text-3xl` from styles\fonts.scss

File: components/CustomInput/index.tsx [hunk 6]
```diff
@@ -28,8 +29,8 @@ const CustomInput: FC<InputProps> = ({
   children,
 }) => {
   return (
-    <div className={styles.customInput}>
-      <label htmlFor={id} className={styles.labelInput}>
+    <div className={styles['custom-input']}>
+      <label htmlFor={id} className={styles['label-input']}>
```
Comment (Eghinner) [inline components/CustomInput/index.tsx]: if restore to camelCase dont need change this file

File: components/ProfileForm/index.tsx [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+              placeholder="Inserta el enlace de tu catálogo web"
+              id="title"
+              type="text"
+              defaultValue="prueba"
+            />
+          </div>
+        </div>
+      </section>
+      <section className={styles['buttons-container']}>
+        <Button iconName="" className={styles['button-cancel']} onClick={() => router.push('/')}>
```
Comment (Eghinner) [inline components/ProfileForm/index.tsx]: better to go back to products

File: components/CustomInput/CustomInput.module.scss (hunk 5 above)
Comment (JuanParraIV) [reply components/CustomInput/CustomInput.module.scss]: i change for this 
image

File: components/ProfileForm/ProfileForm.module.scss (hunk 1 above)
Comment (Israel-Laguan) [reply components/ProfileForm/ProfileForm.module.scss]: I went ahead and made this change

**Comment by Israel-Laguan** [comment]
Tested on local.

It will require more work to add functionality but that can be handled in another PR.

Added active state for new links

**Comment by Israel-Laguan** [comment]
Tested on deployment
image
image

Would need a little bit of more work visually but will merge to fix that as Bugs

**Comment by angelik0828** [comment]
## ❌ The create product button is not there validate what happened

imagen

**Pull request merged**